### DISCUSSION TITLE
Pricing Data

### DISCUSSION THREAD

**Discussion by cskolaut** [peter-oroszvari/tastytrade-api#10] [Ideas]
Peter - thank you for your work on this API. Have you given any thought into being able to get bid/mid/ask pricing on options chains, or pricing for instruments via the DXFeed? It is too bad TastyTrade is not making this available via their endpoints.

**Comment by Graeme22** [comment]
Hi, this is available with the Streamer class, see the documentation here: https://tastyworks-api.readthedocs.io/en/latest/?badge=latest
It's a different package, but the same API, so the Quotes work the same in both places
